src/flashlsAudioTracks.js

```diff
@@ -0,0 +1,38 @@
+import videojs from 'video.js';
+
+/**
+ * Updates the selected index of the audio track list with the new active track
+ *
+ * @param {Object} tech
+ *        The flash tech
+ * @function updateAudioTrack
+ */
+export const updateAudioTrack = (tech) => {
+  tech.audioTracks().tracks_.forEach((track) => {
```

Comment:
You can access the audio tracks within `tech.audioTracks()` with indexing. Instead of doing a `forEach` on the private `tracks_` property, you should just do a normal `for` loop over audio tracks
e.g.
```
const audioTracks = tech.audioTracks();

for (let i =0; < audioTracks.length; i++) {
   ......
}
```